### DISCUSSION TITLE
无法授权ios设备专业版

### DISCUSSION THREAD

**Discussion by skro789** [getlantern/lantern-forum-cn#1085] [General]
![上传1.png...]（）使用授权码的方式，将手机上的动态授权码输入到电脑上显示成功，但实际上一直没有授权成功。手机无法加入蓝灯专业版。为什么？我怎么样可以让手机加入成功，账号邮箱是caroljcr@163.com，希望回复。！[上传图像.png...]（）![IMG_0238]（https://github.com/user-attachments/assets/22138c95-c462-4f15-9ed6-c12048ee9925）![上传IMG_0245![IMG_0246]（https://github.com/user-attachments/assets/523df6e4-0b4e-43f6-ab32-72b765051b89）.PNG...]（）

**Comment by skro789** [comment]
![IMG_0238](https://github.com/user-attachments/assets/c5ac77f6-c441-4645-b4eb-0a62c8b357a3)
![IMG_0245](https://github.com/user-attachments/assets/cf50dc52-156f-4504-988e-426711a304e8)
![Uploading 1.png…]()
![IMG_0246](https://github.com/user-attachments/assets/6e60ccc5-5a48-4d27-888d-4c02a6a130a5)

**Comment by skro789** [comment]
![Uploading 1.png…]()

**Comment by MaxMKMax** [comment]
@skro789 您好. 非常抱歉. iOS专业版的问题一直还没处理好呢. 因为苹果不批准. 您用别的手机或电脑登入专业版ok? 谢谢. 抱歉啊!!!
